_posts/platform/app/2000-01-01-tasks.md

```diff
@@ -55,6 +55,8 @@ $ scalingo --app my-app run bash
 [10:45] Scalingo:my-app ~ $ mongo $SCALINGO_MONGO_URL
 ```
 
+You can also add a shell script that is executed at each start of a new one-off. You need to name it `.profile` and place it at the root of your application.
```

Comment:
Actually it's not yet implemented IIRC

Do not merge before https://github.com/Scalingo/appsdeck-builder/issues/198 is in prod

Plus, it's not for each new one-off, it's for ALL types of containers.